src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Reload/ReloadableProject.cs

```diff
@@ -128,8 +136,37 @@ public Task Initialize()
             // releases its hold on the UI thread. This prevents unnecessary race conditions and prevent the user
             // from interacting with the project until the evaluation has completed.
             await _projectVsServices.ProjectTree.TreeService.PublishLatestTreeAsync(blockDuringLoadingTree: true).ConfigureAwait(false);
-            
+
             return ProjectReloadResult.ReloadCompleted;
         }
+
+        private static bool NeedsForcedReload(ImmutableArray<ProjectPropertyElement> oldProperties, ImmutableArray<ProjectPropertyElement> newProperties)
```

Comment:
This is unneeded coupling to this class. It shouldn't know anything about target frameworks. I'd like to see this factored out.